react-vis-force/build.boot

```diff
@@ -0,0 +1,32 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "0.1.4")
+(def +version+ (str +lib-version+ "-0"))
+
+(task-options!
+ pom  {:project     'cljsjs/react-vis-force
+       :version     +version+
+       :description "react-vis-force applies the react-vis and d4-style component approach to the d3-force library."
+       :url         "https://github.com/uber/react-vis-force"
+       :scm         {:url "https://github.com/cljsjs/packages"}
+       :license     {"BSD" "https://opensource.org/licenses/BSD-3-Clause"}})
+
+(require '[clojure.java.io :as io])
+
+(deftask package []
+  (comp
+    (download :url (str "https://unpkg.com/react-vis-force@" +lib-version+ "/dist/react-vis-force.js")
+              :checksum "5860feef46d67e9b73a3171c0ea51294")
+    (download :url (str "https://unpkg.com/react-vis-force@" +lib-version+ "/dist/react-vis-force.min.js")
+              :checksum "b7fbfc6ecbabef2dc68c995c040f1c80")
+    (sift :move {#"^react-vis-force.js"      "cljsjs/development/react-vis-force.inc.js"
```

Comment:
Add package name to the path: `cljsjs/react-vis-force/development/react-vis-force.inc.js`.